micrometer-tracing/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
 
 dependencies {
 	api 'io.micrometer:micrometer-observation'
+	api 'io.micrometer:context-propagation'
```

Comment:
Is it needed as `api` or `implementation`? In other words, do users of micrometer tracing need context-propagation on the classpath when compiling or only at runtime? If context-propagation types are not in any of tracing's API that users would compile against, then it can be `implementation` instead, I think.

Reply:
ok I'll change it to `implementation` then